actions/post-preview-urls/index.test.ts

```diff
@@ -399,4 +406,48 @@ describe(`Post Preview URLs action`, () => {
             )
         }
     )
+
+    test(
```

Comment:
Just a few suggestions

- Use the BDD Given When Then [Gherkin approach](https://cucumber.io/docs/gherkin/reference/) -- there's even a library for it, which I found pretty cool [https://www.npmjs.com/package/@cucumber/cucumber]
- Pull into dedicated describe, `postPreviewUrls`
- Use a `beforeEach` to wrap the reset
- Add a few comments on expected behavior 

```suggestion
describe('postPreviewUrls', () => {
    const token = '1234';
    let mockRequest: jest.Mock;

    beforeEach(() => {
        mockRequest = jest.fn().mockResolvedValueOnce({
            data: { body: 'Hello World!\n Some indent' }
        });
        mockedGithub.getOctokit.mockReturnValue({ request: mockRequest } as any);
    });

    test(
        strip`
        Given there is no app icon
        When postPreviewUrls is called
        Then no space should be added before the app name
        `,
        async () => {
            // Given
            const previewLinks = JSON.stringify([
                { name: 'Latest', url: 'https://feature.app.example.com' }
            ]);
            const appName = 'App';
            const repo = { owner: 'my-org', repo: 'my-repo' };
            const prNumber = 1;

            // When
            await postPreviewUrls({
                linksJSON: previewLinks,
                token,
                repo,
                prNumber,
                appName
            });

            // Then
            expect(mockedGithub.getOctokit).toHaveBeenCalledWith(token);
            expect(mockRequest).toHaveBeenLastCalledWith(
                'PATCH /repos/{owner}/{repo}/pulls/{pull_number}',
                {
                    body: strip`
                    Hello World!
                     Some indent
                    <!--app-preview-urls-do-not-change-below-->
                    ---
                    **App preview links**
                    _Latest_: https://feature.app.example.com
                    <!--app-preview-urls-do-not-change-above-->
                    `,
                    owner: repo.owner,
                    pull_number: prNumber,
                    repo: repo.repo
                }
            );
        }
    );
});

```